src/style.css

```diff
@@ -0,0 +1,84 @@
+/* No passed !!! */
+
+body {
+  margin: 0;
+  font-family: "Roboto", "sans-serif";
+}
+
+button {
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 16px;/* identical to box height */
+  text-align: center;
+  text-transform: uppercase;
+  color: #fff;
+  background: #00acdc;
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+  width: 100%;
+  padding-top: 12px;
+  padding-bottom: 12px;
+  margin-top: 20px;
+}
+
+button:hover {
+  color: #00acdc;
+  background: #fff;
+  box-sizing: border-box;
+}
+
+.card {
+  width: 198px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product {
+  padding: 30px 20px;
+  width: 158px;
+}
+
+.wrap {
+  padding: 11px 15px 15px;
+}
+
+.article {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  margin: 0;
+}
+
+.code {
+  font-size: 11px;
+  line-height: 13px;/* identical to box height */
+  margin: 6px 0 15px;
+}
+
+.review > img {
+  float: left;
+  width: 13px;
+  height: 14px;
+  padding: 0 5px 0 1px;

```

Comment:
the distance between stars is `4px`. Try to figure it out with `margin-right: 4px` 
![image](https://user-images.githubusercontent.com/29413892/56793569-fcb70c00-6814-11e9-8e77-69c0c7d84e6f.png)

Reply:
Make sure the star is centralized vertically and horizontally in its wrapper